WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
+                <code>reduce_vars.py</code>. For an example of this please See
+                <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Getting-Started">Here</a> for documentation.
+                and <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Example-reduction-script">Here</a>
+                for the full script.
+                Autoreduction expects the reduction script to have a <code>main()</code> method
+                that takes in two arguments, <strong>data</strong> and <strong>output</strong>.
+                These are passed in a kwargs but it may be desirable for these to also be accepted from the command line.
+                For an example of how to do this please
+                <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Getting-Started#main">See Here</a>.
+                The reduction script should perform all operations on the provided data file and
+                save the output to the output directory provided to the script. The output will
+                be copied to the cache upon completion. If additional save locations are
+                required these can be returned as a list of system paths from the main method.
+            </p>
+            {% if support_email %}
+                <p>If you feel your script will need to save to additional locations please
+                    <a href="mailto:{{support_email}}">Contact Us</a> to ensure the locations
+                    are accessible from the autoreduction machine.</p>
+            {% endif %}
+        </div>
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="reduction script file location directory">
+        <div class="panel-heading">
+            <h4>Where should I put reduction scripts?</h4>
+        </div>
+        <div class="panel-body">
+            <p>
+                When you have made sure your reduction script is in the correct format and the
+                variables are exposed it must be placed within the autoreduction directory
+                within the user script directory of the appropriate instrument.
+            </p>
+            <p>
+                For example, the reduction script for MERLIN would be located at:
+                <code>\\isis\inst$\NDXMERLIN\user\scripts\autoreduction\reduce.py</code> <br/>
+                and the variables at:
+                <code>\\isis\inst$\NDXMERLIN\user\scripts\autoreduction\reduce_vars.py</code>
+            </p>
+        </div>
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="reduction data file location directory output reduced">
+        <div class="panel-heading" id="reduction_data_location">
+            <h4>Where does my reduced data go?</h4>
+        </div>
+        <div class="panel-body">
+            <p>
+                When a reduction job has completed the reduced data is moved to the CEPH storage
```

Comment:
`When a reduction job has completed, the...`

Reply:
Leaving this one as it's more verbose worded this way.